docker/Dockerfile

```diff
@@ -34,63 +30,73 @@ RUN git config --global http.sslverify false && \
 ## Setup environment for Serialbox
 ##---------------------------------------------------------------------------------
 FROM fv3gfs-environment as fv3gfs-environment-serialbox
-# install dependencies for serialbox
-ENV DEBIAN_FRONTEND=noninteractive
 # set TZ
-ENV TZ=US/Pacific
+ENV DEBIAN_FRONTEND=noninteractive TZ=US/Pacific
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
+# install dependencies for serialbox
 RUN apt-get update && apt-get install -y \
     cmake \
     libssl-dev \
-    libboost-all-dev \ 
     clang \
     clang-format \
     clang-tidy \
     python3-numpy \
     python3-nose \
     python3-sphinx
 
+# Install headers from the Boost library
+RUN wget -q https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.gz && \
+    tar xzf boost_1_74_0.tar.gz && \
+    cd boost_1_74_0 && \
+    cp -r boost /usr/include/ && cd /    
+
 # install serialbox2 from source
 COPY serialbox2 serialbox2
 RUN cd serialbox2 && \
     mkdir build && \
     cd build && \
     cmake -DSERIALBOX_USE_NETCDF=ON -DSERIALBOX_ENABLE_FORTRAN=ON \
       -DSERIALBOX_TESTING=ON  ../ && \
-    make -j8 && \
+    make -j16 && \
```

Comment:
What does this do?

Reply:
increases the number of concurrent build processes used by make.  This can speed up the building if there are enough compilation steps that can be performed in parallel. (There are -I have also increased build concurrency on ESMF, MPICH and FV3 to 24.)